static/app/components/replays/breadcrumbs/replayTimeline.tsx

```diff
@@ -75,7 +66,7 @@ function ReplayTimeline({}: Props) {
       >
         <MinorGridlines durationMs={durationMs} width={width} />
         <MajorGridlines durationMs={durationMs} width={width} />
-        <CompactTimelineScrubber />
```

Comment:
why did we get rid of the `CompactTimelineScrubber`?

Reply:
Oh I renamed CompactTimelineScrubber to TimelineScrubber and got rid of the original TimelineScrubber